pmml_recommendation/databuilder.py

```diff
@@ -0,0 +1,95 @@
+from random import random, randrange, getrandbits, randint, sample 
```

Comment:
I would apply PEP8 formatting (e.g. with `autopep8`). Simply because we might add format check in GH actions in the future.